lua/acf/shared/missiles/asm.lua

```diff
@@ -193,7 +193,7 @@ ACF.RegisterMissile("9M113 ASM", "ATGM", {
 		PropMass		= 0.1,
 		Thrust			= 15000, 	-- in kg*in/s^2
 		FuelConsumption = 0.0007,	-- in g/s/f
-		StarterPercent	= 0.2,
+		StarterPercent	= 0.002,
```

Comment:
Kornets seem to still be affected by the speed boost mechanic making them get launched backwards. Here's some values that give better results (including a muzzle velocity close to the real one):

* Thrust: 15000 => 25000
* FuelConsumption: 0.0007 => 0.001
* DragCoef: 0.005 => 0.002
* DragCoefFlight: 0.05 => 0.02

Reply:
Must have missed this one, will get to it.